examples/kill-pod.js

```diff
@@ -0,0 +1,31 @@
+
+import { sleep } from 'k6';
+import { Kubernetes } from 'k6/x/kubernetes';
+
+function getPodNames(nameSpace, kubernetes) {
+  const result = {}
+  kubernetes.pods.list(nameSpace).forEach(function(pod){
+    result[pod.name] = pod.status.phase
+  })
+  return result
+}
+
+export default function () {
+  const kubernetes = new Kubernetes({
+    // config_path: "/path/to/kube/config", ~/.kube/config by default
+  })
+  const nameSpace = "default"
+  let podsList = getPodNames(nameSpace, kubernetes)
+  const podName = Object.keys(podsList)[0]
+  kubernetes.pods.kill(podName, nameSpace)
+  // TODO: for some reason we can't feel terminating status
```

Comment:
This is determined by checking `pod.object_meta.deletion_timestamp`. If it's not null, it has received a termination instruction. Maybe `kubernetes.pods.isTerminating(pod)` could return a bool based on this condition, as state changes won't be pushed to the js object you already have.